src/core/layout/gel-defaults.js

```diff
@@ -70,7 +72,7 @@ export const config = screen => {
 			},
 		},
 		audio: {
-			group: "topRight",
+			group: verticalTopRight ? "topRightV" : "topRight",
```

Comment:
Yeah maybe just match the order so `topRightVertical` for the config value.

_(This is probably the most pedantic comment I have ever left)_